lib/event_manager_web/types/event.ex

```diff
@@ -11,7 +11,7 @@ defmodule EventManagerWeb.Types.Event do
     field(:title, non_null(:string))
     field(:description, non_null(:string))
     field(:location, non_null(:string))
-    field(:public, :boolean, default_value: false)
+    # field(:public, :boolean, default_value: false)
```

Comment:
Why commenting and not removing this line?

Reply:
Because we'll be adding it back when we add the `Groups` and `Invitations` features so that you can have private events that are not listed.

So we keep the comment as a reminder for now